docs/image_management/upload_images.md

```diff
@@ -9,37 +9,35 @@ description: Upload images to Ibexa DXP to manage them.
 
 ## Manage folders
 
-By default, uploaded files are stored in the Media root folder.
+By default, uploaded files are stored in the Media root folder of the Content Tree.
 If you want to upload them to different location, navigate to the folder you want to upload to.
 
-### Add folders
+You can also [create additional folders](content_tree.md#add-folders) and sub-folders to further organize your images.
 
-1\. Go to **Content** -> **Media**.
+## Add image to create an image asset
 
-2\. In the upper-right corner click **Create content**.
+1\. In the main menu, go to **Content** -> **Media** -> **Images**.
 
-3\. From the available content types, select **Folder**, and click **Create**.
+2\. In the upper-right corner, click **Create content**.
 
-4\. Enter name for the new folder.
-
-5\. (Optional) Add folder description.
-
-6\. Click **Publish**.
+3\. From the available content types, select **Image** and click **Create**.
 
-The newly created folder is now visible in the Content Tree structure.
-You can create subfolders by clicking the folder and repeating the above procedure.
+4\. Provide image title and optionally, a description.
 
-## Add image
+5\. In the **Image** section, click **Upload file** and from the system browser, select an image.
 
-1\. Go to **Content** -> **Media**.
+6\. In the **Alternative text** field, provide an alternative description intended for users with vision impairment.
 
-2\. In the upper-right corner click **Create content**.
+<a name="ai"/> </a>
 
-3\. From the available content types, select **Image** and click **Create**.
+!!! note "Use AI to generate alternative text"
 
-4\. Provide image title.
+    If the AI Actions LTS update has been installed and configured in your application, the **Use AI** button is visible.
+    If you have `Action configuration/View` and `Action configuration/Execute` permissions, you can click it to have the alternative text generated automagically.
```

Comment:
I wouldn't repeat these policies here. It's not that important for the end user, and adds details we need to maintain.